app/src/main/java/org/ole/planet/myplanet/ui/exam/TakeExamFragment.java

```diff
@@ -78,20 +78,22 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         initExam();
         questions = mRealm.where(RealmExamQuestion.class).equalTo("examId", exam.getId()).findAll();
-        tvQuestionCount.setText("Question : 1/" + questions.size());
+        boolean hasQuestions = questions.size() > 0;
+        tvQuestionCount.setText("Question : " + (hasQuestions ? "1" : currentIndex) +" /" + questions.size());
         if (!isMySurvey)
             sub = mRealm.where(RealmSubmission.class)
                     .equalTo("userId", user.getId())
                     .equalTo("parentId", exam.getId())
                     .sort("startTime", Sort.DESCENDING)
                     .equalTo("status", "pending")
                     .findFirst();
-        if (questions.size() > 0) {
+        if (hasQuestions) {
             createSubmission();
             Utilities.log("Current index " + currentIndex);
             startExam(questions.get(currentIndex));
         } else {
             container.setVisibility(View.GONE);
+            btnSubmit.setVisibility(View.GONE);
```

Comment:
Also issue was about not showing `Take Exam` changes here seems to be done to not submit exam.

Reply:
@lmmrssa if there is no any question the container is invisible so there is no point to show the submit answer button.

Reply:
@lmmrssa could you please elaborate the problems what it would supposed to be and what is happened here so would be nice and easier for me to understand and i will update the commit.

Reply:
@yuviii I don't see a case when user will end up in this page if there is no question. We are not showing them **Take Exam** button at all. And on mobile app user will not be able to navigate to pages on their own like on web.

Reply:
@lmmrssa i have hide the button `Taketest` incase if there is no question. Could you share the screenshot where you find the take test?

Reply:
@yuviii That is what i'm saying. There is no Take test so user will not end up coming to exam page at all. So we do not need to add condition in exam page.